src/components/Tabs/Annotation/AnnotationDisplay/ListItems/TrackListItem.tsx

```diff
@@ -3,6 +3,7 @@ import { Color } from "three";
 
 import { Dataset } from "../../../../../colorizer";
 import { FlexRowAlignCenter } from "../../../../../styles/utils";
+import { formatQuantityString } from "../../../../../utils/formatting";
```

Comment:
![image](https://github.com/user-attachments/assets/04afcb1f-8f1b-4331-b91a-867e2bc0baf3)

Added the small label text to the right of the track number

Reply:
stray thought: the many `../../../../ ...`s here makes me think we ought to investigate absolute import paths for this and/or other projects? very very low-priority though